LiteNetLib/Interfaces/INetSocket.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Net;
+
+namespace LiteNetLib.Interfaces
+{
+    internal interface INetSocket : ISendSocketData
+    {
+        int LocalPort { get; }
+        short Ttl { get; set; }
+
+        bool Bind(IPAddress addressIPv4, IPAddress addressIPv6, int port, bool reuseAddress, bool ipv6);
+        void Close();
+        bool SendBroadcast(byte[] data, int offset, int size, int port);
```

Comment:
Maybe move `SendBroadcast(byte[] data, int offset, int size, int port)` to `ISendSocketData` too?

Reply:
I undid some of the interface extractions, so this is no longer relevant.